src/foam/net/web/HTTPRequest.js

```diff
@@ -162,6 +162,7 @@ foam.CLASS({
           this.headers['Content-Type'] = 'application/x-www-form-urlencoded';
           break;
         }
+        this.headers['Content-Type'] += '; charset=utf-8'
```

Comment:
I don't know enough about character encoding to know if I shouldn't do this across all HTTPRequests we send. Do any of you have any thoughts?
I could make this configurable via a property and give it a default value of utf-8, or I can do something different if you have any ideas.

Reply:
Spoke to @adamvy offline and he suggested putting this fix on HTTPBox because we might not always want utf-8 on HTTPRequests but we always want it when talking over the HTTPBox.